File: 自写/Demo/ObjcRuntimeDemo/ObjcRuntimeDemo/Classes/MessageForwarding/MessageForwardingController.m
//
//  MessageForwardingController.m
//  ObjcRuntimeDemo
//
//  Created by luozhijun on 2017/3/16.
//  Copyright © 2017年 RickLuo. All rights reserved.
//

#import "MessageForwardingController.h"
#import "MessageForwarding.h"

@interface MessageForwardingController ()

@end

@implementation MessageForwardingController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    MessageForwarding *mf = [MessageForwarding new];
    [mf action];
}

@end

File: 自写/Demo/ObjectiveCRuntimeDemo/ObjectiveCRutimeDemo/MessageForwarding/MessageForwardingController.swift
//
//  MessageForwardingController.swift
//  ObjectiveCRutimeDemo
//
//  Created by luozhijun on 2017/3/17.
//  Copyright © 2017年 RickLuo. All rights reserved.
//

import UIKit

class MessageForwardingController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let mf = MessageForwarding()
        mf.action()
    }

}
